### DISCUSSION TITLE
3rd option for overlapping keybinds

### DISCUSSION THREAD

**Discussion by Wypixla1** [ppy/osu#25258] [Ideas] [answered]
So far, its just keep existing or keep new, but there should be a third option something along the lines of keep both. There are certain rulesets, custom or not, that this was something I relied on (mostly just Taiko, i cannot for the life of me read without it, and Tau (custom ruleset) which is just unfun without overlapping keybinds) There are also some potential uses in the editor, particularly for people who always use 2 editor features together, whatever they may be.

**Comment by peppy** [comment, marked as answer]
This will not be supported, sorry.
